src/test/java/tests/PreferencesTests.java

```diff
@@ -1,46 +1,64 @@
 package tests;
 
 import org.junit.Test;
-import prefs.GlobalConfig;
+import prefs.SessionConfig;
 import prefs.PanelInfo;
 import prefs.Preferences;
 
+import java.lang.reflect.Field;
 import java.util.ArrayList;
 import java.util.List;
 
-import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.*;
 import static org.mockito.Mockito.*;
 
 public class PreferencesTests {
     /**
-     * Tests that Preferences' clearLastOpenBoard method calls GlobalConfig's clearLastOpenBoard once
+     * Tests that Preferences' clearLastOpenBoard method calls SessionConfig's clearLastOpenBoard once
      */
     @Test
     public void testClearLastOpenBoard() {
-        GlobalConfig config = mock(GlobalConfig.class);
-        Preferences prefs = Preferences.load(Preferences.TEST_CONFIG_FILE);
+        SessionConfig config = mock(SessionConfig.class);
+        Preferences prefs = Preferences.load(Preferences.TEST_SESSION_CONFIG_FILENAME);
 
-        prefs.global = config;
-        prefs.clearLastOpenBoard();
+        try {
+            Field sessionConfigField = Preferences.class.getDeclaredField("sessionConfig");
+            sessionConfigField.setAccessible(true);
+            sessionConfigField.set(prefs, config);
+            prefs.clearLastOpenBoard();
+
+        } catch (IllegalAccessException | NoSuchFieldException e) {
+            e.printStackTrace();
+            fail();
+        }
```

Comment:
Can we abstract the reflection to a method? Since we use it twice. We may need more in the future, also.